tests/test_classifier/test_prcurve.py

```diff
@@ -353,9 +352,10 @@ def test_quick_method(self):
             iso_f1_curves=True,
             ap_score=False,
             random_state=2,
+            show=False
```

Comment:
I think we can remove  the `random_state` param now that it's been removed from the quick method